packages/preview/modern-ecnu-thesis/0.3.0/template/thesis.typ

```diff
@@ -0,0 +1,364 @@
+#import "@preview/modern-ecnu-thesis:0.3.0": documentclass, indent, no-indent, word-count-cjk, total-words
+#import "../utils/bilingual-figure.typ": *
```

Comment:
If you want to use these functions in the template, you should re-export them in `lib.typ` and import them with the rest of the package above.